tools/release_tool/git_release.sh

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+# This script is to automate the process of monthly release with github API
+
+# Test if getopt is enhanced version
+getopt --test > /dev/null
+if [ $? -ne 4 ]; then
+  echo "[ERROR] Your system doesn't have enhanced getopt"
+  echo 2
+fi
+
+function Usage()
+{
+  echo "Usage: ./$(basename ${BASH_SOURCE[0]}) --tag TAG --release_note RELEASE_NOTE \
+--token TOKEN [--release_name RELEASE_NAME] [--commitish COMMITISH] [--draft] \
+[--host_name HOST_NAME] [--repo_owner REPO_OWNER] [--repo_name REPO_NAME] [--asset] ..."
+  echo ""
+  echo "[OPTIONS]"
+  echo "--tag              The name of the tag"
+  echo "--release_name     The name of the release"
+  echo "--release_note     Path of text file describing the contents of the release"
+  echo "--commitish        The commitish value that determines where the Git tag is created from"
+  echo "--draft            Create a draft release"
+  echo "--token            User token for authentication"
+  echo "--host_name        Host name for endpoint URL [Enterprise-specific endpoint only]"
+  echo "--repo_owner       Owner of the repository"
+  echo "--repo_name        The name of the repository"
+  echo "--asset            Path of release asset"
+  echo ""
+  echo "[EXAMPLE]"
+  echo "$ ./git_release.sh --tag 1.9.0 --commitish release/1.9.0 --token 0de25f1ca5d1d758fe877b18c06 \\"
```

Comment:
Please hide token. (regardless it is not valid)

Reply:
@mhs4670go Is this token invalid? If it is invalid token, I'll approve this PR.

Reply:
@hseok-oh Yes:) That is dummy token.